lib/SPIRV/SPIRVReader.cpp

```diff
@@ -4262,6 +4262,20 @@ bool SPIRVToLLVM::transFPGAFunctionMetadata(SPIRVFunction *BF, Function *F) {
     MetadataVec.push_back(ConstantAsMetadata::get(getUInt32(M, Literals[1])));
     F->setMetadata(kSPIR2MD::LoopFuse, MDNode::get(*Context, MetadataVec));
   }
+  if (BF->hasDecorate(internal::DecorationMathOpDSPModeINTEL)) {
+    std::vector<Metadata *> MDVecDSPPref;
+    auto Literals =
```

Comment:
Could you please not use `auto` here? It is not obvious what return type `getDecorationLiterals` has.